.travis.yml

```diff
@@ -3,7 +3,7 @@ sudo: required
 dist: trusty
 addons:
   ssh_known_hosts: localhost
-  postgresql: "9.5"
```

Comment:
prettier!

Reply:
Yes, I need to fix my prettier config :)